Added card effect text background

LokiMidgard opened the pull request:
I added a background texture to the cards:

![image](https://user-images.githubusercontent.com/389101/102077300-595fa400-3e09-11eb-816e-8156306d4d16.png)

I also replaced the background color of threshold and target with an textures.

Major and Minor cards now also have a coressponding border:

![image](https://user-images.githubusercontent.com/389101/102099239-6049df00-3e28-11eb-8b52-1926a632043e.png)

LokiMidgard committed:
added normal background

LokiMidgard committed:
Added padding

LokiMidgard committed:
Added Overlay for threshold and target

LokiMidgard committed:
Added border for major and minor cards

Gudradain commented:
I really like the change. Only 2 minor details:

1. I would prefer if minor and major modified the card background image rather than using :after as it breaks the "inspect" when looking at the code in the browser.
2. For the round image in the bottom right corner of the rules text, it would be best if the :after was smaller in size and fixed to the bottom right in order to not break the "inspect".

In any case, I'm going to merge this pull request later. The only part that I will remove is the minor/major for now.

LokiMidgard commented:
I'm not sure what you mean when you say
> breaks the "inspect"

when I us the inspect function in the browser it shows `:after` and `:before` without problems.

Putting this in the background will make the HTML easier since no overlay for the cost is needed. But then you would 4 different images. With using the overlay for cost, it could actually go down to one.

LokiMidgard committed:
changed image size of rules container overlay

LokiMidgard committed:
removed minor/major border

LokiMidgard commented:
Ok I think I get now what you mend.

I fixed the image size on the rule-text overlay.

I also removed the minor major border. But I hope there is a way to get the border back without a drawback.

E.g. instead of adding the border over every thing the boarder could be a normal element in the HTML and the content of the card is just in that HTML element. Just the background must be visible outside of that border... 🤔 

But that would propably a different pull request.
